Localization/en-US_Mods.MechTransfer.hjson

```diff
@@ -0,0 +1,315 @@
+Items: {
+	PneumaticActuatorItem: {
+		DisplayName: Pneumatic Actuator
+		Tooltip: Used to craft item transfer devices
+	}
+
+	BigButtonItem: {
+		DisplayName: Big Button
+		Tooltip: ""
+	}
+
+	SmallButtonItem: {
+		DisplayName: Small Button
+		Tooltip: ""
+	}
+
+	OmniTurretItem: {
+		DisplayName: Omni Turret
+		Tooltip: Shoots any standard ammo
+	}
+
+	SuperOmniTurretItem: {
+		DisplayName: Super Omni Turret
+		Tooltip: Shoots any standard ammo
+	}
+
+	MatterProjectorItem: {
+		DisplayName: Matter Projector
+		Tooltip: Shoots any standard ammo really, really fast
+	}
+
+	TransferPipeItem: {
+		DisplayName: Transfer Pipe
+		Tooltip: Used to connect item transfer devices
+	}
+
+	TransferExtractorItem: {
+		DisplayName: Transfer Extractor
+		Tooltip: Extracts items from adjacent chests
+	}
+
+	StackExtractorItem: {
+		DisplayName: Stack Extractor
+		Tooltip: Extracts a whole stack at once
+	}
+
+	TransferInletItem: {
+		DisplayName: Transfer Inlet
+		Tooltip: Picks up dropped items
+	}
+
+	TransferFilterItem: {
+		DisplayName: Transfer Filter (Whitelist)
+		Tooltip: ""
+	}
+
+	InverseTransferFilterItem: {
+		DisplayName: Transfer Filter (Blacklist)
+		Tooltip:
+			'''
+			Place in line with Transfer pipe
+			Right click with item in hand to set filter
+			'''
+	}
+
+	TransferGateItem: {
+		DisplayName: Transfer Gate
+		Tooltip: Place in line with Transfer pipe to toggle the item flow with wire
+	}
+
+	TransferRelayItem: {
+		DisplayName: Transfer Relay
+		Tooltip: Receives items, and sends them out again
+	}
+
+	TransferInjectorItem: {
+		DisplayName: Transfer Injector
+		Tooltip: Injects items into adjacent chests
+	}
+
+	TransferOutletItem: {
+		DisplayName: Transfer Outlet
+		Tooltip: Drops item
+	}
+
+	MagicStorageInterfaceItem: {
+		DisplayName: Magic Storage Interface
+		Tooltip: Allows you to inject and extract items from storage systems
+	}
+
+	MagicStorageExtraInterfaceItem: {
```

Comment:
AFAIK, MagicStorageExtra doesn't exist anymore. Consider removing this, and also any other references to MagicStorageExtra (MagicStorageExtraInterfaceTile.cs, the corresponding line in content.json, and maybe others)

Reply:
Deleted anything with MagicStorageExtra in it. Now it's only mentined in description.txt and README.md. It's probably best if you handle those two